test/unit/formats/odata.js

```diff
@@ -930,6 +930,92 @@ describe('odata message composition', () => {
         });
       });
 
+      describe('with $skiptoken', () => {
+        const billy = { __id: 'cf9a1b5cc83c6d6270c1eb98860d294eac5d526d', age: 4, name: 'Billy' };
+        const blain = { __id: 'c76d0ccc6d5da236be7b93b985a80413d2e3e172', age: 6, name: 'Blaine' };
+
+        [
+          {
+            $top: 0,
+            skiptoken: { repeatId: billy.__id },
+            expectedNext: false,
+            expectedValue: [],
+          },
+          {
+            $top: 1,
+            skiptoken: { repeatId: billy.__id },
+            expectedNext: billy.__id,
```

Comment:
It seems surprising that the nextLink actually points to the (last) value just returned.  Is this expected?